src/integrationTest/java/org/opensearch/test/framework/testplugins/dummy/dummyaction/DummyRequest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.test.framework.testplugins.dummy.dummyaction;
+
+import org.opensearch.action.ActionRequest;
+import org.opensearch.action.ActionRequestValidationException;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.xcontent.ToXContent;
+import org.opensearch.core.xcontent.XContentBuilder;
+
+import java.io.IOException;
+
+public class DummyRequest extends ActionRequest implements ToXContent {
+
+    private final String message;
+
+    public DummyRequest(final StreamInput in) throws IOException {
+        super(in);
+        message = in.readString();
+    }
+
+    public DummyRequest(String message) {
+        this.message = message;
+    }
+
+    /**
+     * @return
```

Comment:
Unfinished doc?

Reply:
This doesn't need any java doc. I'll remove it